docs/installing/prerequisites.md

```diff
@@ -28,8 +28,8 @@ capabilities:
 
 * Manage eBPF maps limits (`CAP_SYS_RESOURCE`)
 * Load and Attach eBPF programs:
-    1. `CAP_BPF`+`CAP_PERFMON` for recent kernels (>=5.8)
-    2. or `CAP_SYS_ADMIN` for older kernels
+    1. `CAP_BPF`+`CAP_PERFMON` for recent kernels (>=5.8) where the kernel perf paranoid value in `/proc/sys/kernel/perf_event_paranoid` is equal to 1 or less
```

Comment:
Above it seems that you check if paranoid level is more than 3, not 1 like written here. Which one is true?

Reply:
1, forgot to update the documentation, sorry